games-engines/odamex/odamex-0.8.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils desktop wxwidgets xdg
+
+DESCRIPTION="Online multiplayer, free software engine for Doom and Doom II"
+HOMEPAGE="https://odamex.net/"
+SRC_URI="mirror://sourceforge/${PN}/Odamex/${PV}/${PN}-src-${PV}.tar.bz2"
+
+LICENSE="GPL-2+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client master upnp +odalaunch portmidi server"
```

Comment:
Could you sort this? upnp to last. Ebuild looks good otherwise. Would you like to become a proxy-maintainer for this?

Reply:
Sure, I can proxy-maintain this package. I've gone ahead and replaced Mike Wood with myself in the metadata.xml file.